docs/_layouts/post.html

```diff
@@ -10,87 +10,29 @@
   <div class='timestamp left text-center'>
     <span class='month'>{{ page.date | date: "%h" }}</span>
     <span class='date'>{{ page.date | date: "%d" }}</span>
+    <!--  Reuse the month formatting to add the year to the post-->
+    <span class='month'>{{ page.date | date: "%Y" }}</span>
   </div>
   <div class='info'>
-    <h3>
-      <a href='{{page.url}}'>{{ page.title }}</a>
+    <h3 class='regularText'>
+      {{ page.title }}
```

Comment:
No, we do not need this. You can add the special characters to the title in the post's title field and they will be displayed.